cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -40,14 +40,14 @@ To use the **Docker image**:
 
 - Once Docker has started, pull down the *CDAP Docker Image* from the Docker hub using::
 
-    $ docker pull caskdata/cdap-standalone
+    $ docker pull caskdata/cdap-standalone:\ |literal-release|
```

Comment:
It does not.

http://docs-staging.cask.co/cdap/3.0.1-feature-docker-pull-version/en/developers-manual/getting-started/standalone/docker.html